IBPSA/Fluid/BaseClasses/FlowModels/basicFlowFunction_inv.mo

```diff
@@ -0,0 +1,43 @@
+within IBPSA.Fluid.BaseClasses.FlowModels;
```

Comment:
I think that we can add this function in the inverse annotation of `basicFlowFunction_m_flow` and `basicFlowFunction_dp`. In that sense, a more consistent naming would be `basicFlowFunction_k`.
Vice versa, `basicFlowFunction_m_flow` and `basicFlowFunction_dp` can be added in the inverse annotation of this function. This would however require the model to be refactored to have `m_flow_turbulent` as an input instead of `m_flow_small`. For I consistency reasons I believe that it is worth considering this change.

Reply:
I wonder about the renaming because `basicFlowFunction_m_flow` outputs `dp`, not `m_flow`. So the name of the existing flow functions refers to the input variable, not to the output variable. With this logic the function should be renamed `basicFlowFunction_m_flow_dp`.

About the inverse, please note that currently the function only represents the turbulent regime. This is because it is the regime where the actual opening matters for any control sequence that would use it (also because the condition to transition to laminar flow is debatable for dampers and valves as it does not take into account the variation of the free area). 

Representing the laminar regime would lead to something like
```
k := smooth(2, if m_flow > m_flow_turbulent then
    mPos_flow / dpSqrt else
    0.866025403784439 * sqrt(-mPos_flow * (1/6 * mPos_flow^4 -
    mPos_flow^2 * m_flow_turbulent^2 - 0.5 * m_flow_turbulent^4)) /
    dpSqrt / sqrt(m_flow_turbulent^3));
```
Please let me know if this is worth considering.

Reply:
Indeed, there are some trade-offs to be considered here. @mwetter what are your thoughts on this?

Reply:
The transitioning to laminar -- which is really in our implementation simply a regularization that ensures that the derivative is bounded near zero -- is needed only for numerical reasons. If for this model you can guarantee that the derivative is bounded then we won't need this transition as laminar flow through a damper is not likely happening.

I think clearest would be to call this function `basicFlowFunction_m_flow_dp` as this denotes its input arguments.